### PR TITLE
ipwcli_dns: fixed markups in doc

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Fixed references from C(param=value) to I(param=value).

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ipwcli_dns

### CODE REVIEW

File: plugins/modules/net_tools/ipwcli_dns.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
     address:
         description:
             - The IP address for the A or AAAA record.
-            - Required for C(type=A) or C(type=AAAA)
+            - Required for I(type=A) or I(type=AAAA)
```
Comment (felixfontein) [inline plugins/modules/net_tools/ipwcli_dns.py:48]: Do you mind adding a period after the sentences as well, while we're at this? Same for the others as well :)
```suggestion
            - Required for I(type=A) or I(type=AAAA).
```

**Review by felixfontein — COMMENTED** [review]
:heart:

File: plugins/modules/net_tools/ipwcli_dns.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/net_tools/ipwcli_dns.py]: done

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for improving the docs!

**Pull request merged**